includes/class-wc-stripe-webhook-handler.php

```diff
@@ -692,8 +697,13 @@ public function process_review_opened( $notification ) {
 
 		$order->update_meta_data( '_stripe_status_before_hold', $order->get_status() );
 
-		/* translators: 1) The URL to the order. 2) The reason type. */
-		$message = sprintf( __( 'A review has been opened for this order. Action is needed. Please go to your <a href="%1$s" title="Stripe Dashboard" target="_blank">Stripe Dashboard</a> to review the issue. Reason: (%2$s)', 'woocommerce-gateway-stripe' ), $this->get_transaction_url( $order ), $notification->data->object->reason );
+		$message = sprintf(
```

Comment:
This can be tested by creating an order, pay it with card, set it to processing again. And then use this postman endpoint to execute a webhook
[Stripe.postman_collection2.json.zip](https://github.com/woocommerce/woocommerce-gateway-stripe/files/7629048/Stripe.postman_collection2.json.zip)

Remember to change the charge to the one of the new order. You can find it in wp_post_meta with name _transaction_id. After that the string will show up as a note inside the order